umbraco/Umbraco.CMS.Backoffice
# Feature/blueprint

Rename "Content Template" to "Document Blueprint" to be consistent in the CMS.

## Description

Previously Content Template and Document Blueprint were different names in the backoffice for the same feature. Renamed Content Template to be Document Blueprint for naming consistency in the CMS.

## Types of changes




## Motivation and context

The naming for the same feature was different in the Content and Settings section.

## How to test?

Right click on a content node in the content section and verify the Document Blueprint name. All notifications should also reflect Document Blueprint and the settings tree.

## Screenshots (if appropriate)

## Checklist
